docs/harbor_compatibility_list.md

```diff
@@ -0,0 +1,51 @@
+
+## Replication Adapter
+
+|     | Registries       | Pull Mode | Push Mode | Introduced in Release | Automated Pipeline Covered |
+|-----|------------------|-----------|-----------|-----------------------|---------------------------|
+| Harbor|  ![Harbor](img/replication_adapters/harbor_logo.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | Y |
+| distribution | ![distribution](img/replication_adapters/distribution.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | Y |
+| docker hub | ![docker hub](img/replication_adapters/docker_hub.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | Y |
+| Huawei SWR | ![Huawei SWR](img/replication_adapters/hw.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | N |
+| GCR | ![GCR](img/replication_adapters/gcr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | Y |
+| ECR | ![ECR](img/replication_adapters/ecr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | Y |
+| ACR | ![ACR](img/replication_adapters/acr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | N |
+| AliCR | ![AliCR](img/replication_adapters/ali-cr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | N |
+| Helm Hub | ![Helm Hub](img/replication_adapters/helm-hub.png)|![Y](img/replication_adapters/right.png)| N/A | V1.9 | N |
+| Artifactory | ![Artifactory](img/replication_adapters/artifactory.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+| Quay | ![Quay](img/replication_adapters/quay.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+| GitLab Registry | ![GitLab Registry](img/replication_adapters/gitlab.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+
+**Notes**: 
+
+* `Pull` mode is replicating artifacts from specified source registries into Harbor. 
+* `Push` mode is replicating artifacts from Harbor to specified target registries.
+
+## OIDC Adapter
+
+|   |  OIDC Providers | Officially Verified | End User Verified   | Verified in Release |
+|---|-----------------|---------------------|---------------------|-----------------------|
+| Google Identity | ![google identity](img/OIDC/google_identity.png)| ![Y](img/replication_adapters/right.png) |  |V1.9|
+| Dex | ![dex](img/OIDC/dex.png) | ![Y](img/replication_adapters/right.png)| | V1.9 |
+| Ping Identity | ![ping identity](img/OIDC/ping.png) | | ![Y](img/replication_adapters/right.png)| V1.9 |
+| Keycloak | ![Keycloak](img/OIDC/keycloak.png) | ![Y](img/replication_adapters/right.png) | | V1.10 |
+| Auth0 | ![Auth0](img/OIDC/auth0.png) | ![Y](img/replication_adapters/right.png) | | V1.10 |
+
```

Comment:
Could we provide links in the first column?

Reply:
added